docs/docs/overview/quickstart/get-started-with-node.js.md

```diff
@@ -0,0 +1,244 @@
+
+<aside>
+🔑  **Note:** To make all of your app users subscribers, you need to programmatically add them to Novu.
+
+</aside>
+
+## Trigger A Notification
+
+Copy and paste the following code into your app to trigger a notification:
+
+```jsx
+const notificationTemplateId = 'first-email';
+
+novu.trigger(notificationTemplateId, {
+  to: {
+    subscriberId: '7789',
+  },
+  payload: {},
+});
+```
+
+Before running the code, make sure you understand the following:
+
+- The value of `notificationTemplateId` should be the notification template’s trigger ID/slug.
+
+![trigger_id.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1686776585/trigger_id_xkhsx7.png)
+
+- The value of `payload` is an array of the data that you want to be dynamically injected into the notification template content.
+- The value of `subscriberId` is the id of the subscriber on Novu. Replace `7789`
+
+with your subscriber ID.
+
+Run the code to trigger a notification!
+
+```bash
+node index.js
+```
+
+# Topics
+
+Novu provides a simple API that offers an easy interface for triggering notifications to multiple subscribers at once. This API is called "Topics" and allows users to manage their bulk notifications without having to implement complex loops. A topic is identified by a custom key that is provided by the user, and this key will be the identifier used in the Topics API.
+
+<aside>
+⚠️ DANGER: The topic key should be unique and can't be changed once chosen. Novu also safe guards for key uniqueness behind the scenes.
+
+</aside>
+
+Users can assign a name to a topic for descriptive purposes. This name does not need to be unique and can be changed using the API.
+
+A topic can have different subscribers, who will receive a notification whenever a notification is sent to the topic.
+
+## **Create a topic[](https://docs.novu.co/platform/topics#create-a-topic)**
+
+Copy and paste the following code into your app to create a topic:
+
+```jsx
+const result = await novu.topics.create({
+  key: 'unique-topic-identifier',
+  name: 'descriptive-topic-name',
+});
+```
+
+Before running the code, make sure you understand the following:
+
+- When creating a `key`, ensure it is unique and accurately identifies the topic. Document naming conventions and communicate them to team members to avoid confusion and ensure a smooth workflow.
+- The value of `name` should be a descriptive topic name.
+
+## Add subscribers to a topic\*\*
+
+Copy and paste the following code into your app to add subscribers to a topic.
+
+```jsx
+const topicKey = '<YOUR TOPIC KEY>';
+
+const response = await novu.topics.addSubscribers(topicKey, {
+  subscribers: ['subscriber-id-1', 'subscriber-id-2', ...],
+});
+```
+
+- To add multiple subscribers to a topic, simply separate their names with commas and add them to the array as part of the `subscribers` value.
+
+## **Sending a notification to a topic[](https://docs.novu.co/platform/topics#sending-a-notification-to-a-topic)**
```

Comment:
Is there a particular reason you are linking here? We never link headers. 

We can link somewhere in the body.

Reply:
`## Send a notification to a topic`

Reply:
I don't recall any particular reason, anyway, it's removed and switched with your suggestion.